packages/lib/src/toolbar/controls/NotationToggleGroup.tsx

```diff
@@ -0,0 +1,34 @@
+import { Notation } from '../../vis/matrix/models';
+import styles from './NotationToggleGroup.module.css';
+import ToggleGroup from './ToggleGroup';
+
+interface Props {
+  value: Notation;
+  onChange: (val: Notation) => void;
+}
+
+const ID = 'h5w-matrix-notation';
+
+function NotationToggleGroup(props: Props) {
+  const { value, onChange } = props;
+
+  return (
+    <div className={styles.root}>
+      <label className={styles.label} htmlFor={ID}>
+        Notation
+      </label>
```

Comment:
Should the label be a feature of `ToggleGroup` like `Selector`? It can wait until we need a label on another toggle group, though.

Reply:
Agreed.